src/spetlr/delta/delta_handle.py

```diff
@@ -25,22 +28,49 @@ class DeltaHandleInvalidFormat(DeltaHandleException):
 
 
 class DeltaHandle(TableHandle):
-    def __init__(self, name: str, location: str = None, data_format: str = "delta"):
+    def __init__(
+        self,
+        name: str,
+        location: str = None,
+        data_format: str = "delta",
+        options_dict: dict = None,
```

Comment:
the stream_start:datetime is another property that the configurator cannot support. I suggest that you change this to a stream_start:Union[datetime,str] where you write in the docstring that you can accept anything that the [dateparser](https://github.com/scrapinghub/dateparser) library can eat.

Reply:
Implemented